### PR TITLE
[Misc] Ignore only title translation in XWiki XML files

### PR DESCRIPTION
**Pull request by surli**
@atallahade @tmortagne This enforces the behaviour we talked about in https://github.com/xwiki/xwiki-platform/pull/1042

### CODE REVIEW

File: weblate-scripts/pre_commit.py [hunk 1]
```diff
@@ -46,11 +46,15 @@ def properties_to_xwiki_xml(file_path, path_prefix, lang):
     if not properties.is_empty():
```
Comment (atallahade) [inline weblate-scripts/pre_commit.py]: I think you can remove this condition.

Comment (surli) [reply weblate-scripts/pre_commit.py]: it might simplify it, right.

**Comment by atallahade** [comment]
It actually kind of makes sense to ignore translations without contents (even if the title is set) :+1:

**Comment by surli** [comment]
Fixed.

**Pull request merged**